VERSION

```diff
@@ -0,0 +1 @@
+v0.0.8
```

Comment:
This is a duplication of the version in the `superduperdb` package. We need to implement a feature which
pulls this version into the python code. This will require creating a `MANIFEST.in` file and fetching the version our in `superduperdb.__init__.py`.

See here: https://github.com/SuperDuperDB/superduperdb/blob/main/superduperdb/__init__.py.

Reply:
Tha Makefile is configured to read the target version from `VERSION` file and then update `__init__.py` accordingly